aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -84,10 +89,17 @@ def export_zip(what, outfile='testzip', overwrite=False, silent=False, use_compr
     if not overwrite and os.path.exists(outfile):
         raise exceptions.ArchiveExportError("the output file '{}' already exists".format(outfile))
 
-    time_start = time.time()
-    with ZipFolder(outfile, mode='w', use_compression=use_compression) as folder:
-        export_tree(what, folder=folder, silent=silent, **kwargs)
     if not silent:
+        file_format = 'Zip (compressed)' if use_compression else 'Zip (uncompressed)'
+        print_header(file_format, outfile, debug, **kwargs)
+
+    if debug:
```

Comment:
how about moving anything that is not specific to export_zip outside?
e.g. timing should probably be handled at the top level, same as "File written"...

Reply:
Not really related to the progress bar - but sure 👍 I think several of your comments are not related to the progress bar, but since many of my changes to the code is also not directly related to this, it seems completely fair :)